.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    working_directory: ~/repo
+    steps:
+      - checkout
+
+      - run:
+        name: Decode Env File
+        command: echo ${ENV_FILE} | base64 --decode > .env
+
+      - run:
+        name: Build Docker
+        command: docker-compose build
+
+  deploy:
+    machine: true
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+        name: Decode Env File
+        command: echo ${ENV_FILE} | base64 --decode > .env
+
+      - run:
+        name: Deploy container
+        command: |
+          scp -r . ${DROPLET_USER}@${DROPLET_IP}:/server
+          ssh ${DROPLET_USER}@${DROPLET_IP} "cd /server && docker-compose up"
```

Comment:
no need to `docker-compose down`? also, shouldn't be `docker-compose up  -d` to get it on the background?